Automattic/jetpack
# Protect: Add activation and deactivation hooks

#### Changes proposed in this Pull Request:

* Redirect to plugin page at activation
* Add link to plugin's page on plugin listing
* remove connection on deactivation

#### Jetpack product discussion


NA

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure all Jetpack plugins are deactivated
* Activate Protect
* Confirm you got redirected to the Protect plugin page
* Set up Protect
* Go to Plugins 
* Confirm there's a link to "Dashboard" that links to the Protect page
* Deactivate Protect
* Activate Protect
* Confirm that you have to Set up Protect again (the connection was lost when you deactivated)
* Activate another plugin, like Boost
* Deactivate Protect
* Make sure Boost is still conected
*


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202161351799997
